shelley/chain-and-ledger/shelley-spec-ledger-test/src/Test/Shelley/Spec/Ledger/Utils.hs

```diff
@@ -147,7 +147,7 @@ mkGenKey seed =
    in (sk, VKey $ deriveVerKeyDSIGN sk)
 
 -- | For testing purposes, generate a deterministic key pair given a seed.
-mkKeyPair ::
+mkKeyPair :: forall era kd.
```

Comment:
we're in the habit of omitting `forall` quantifiers (unless we need them) - there are quite a few in this PR that we could remove

Reply:
When a function has a type that might require using a type application (which is the purpose of this PR) it is important that we know the order of the variables. This function must have the era as its first type parameter. Without a type application fixing era, every function that calls this becomes ambiguous.